test/jdk/java/awt/datatransfer/HTMLDataFlavors/ManualHTMLDataFlavorTest.java

```diff
@@ -90,202 +98,63 @@ public void drop(DropTargetDropEvent dtde) {
-                "   otherwise for instance iexplore can prohibit drag and drop from",
-                "   the browser to other applications because of",
-                "   the protected mode restrictions.",
-                "   On Mac OS X do NOT use Safari, it does not provide the needed DataFlavor",
-                "3) Check the data in the output area of this dialog",
-                "5) The output should not contain information that any of",
-                "   flavors is not present in the system clipboard",
-                "6) The output should contain data in three different formats",
-                "   provided by the system clipboard",
-                "    - Data after the \"ALL:\" marker should include the data",
-                "      from the \"SELECTION:\" marker",
-                "    - Data after the \"FRAGMENT\" marker should include the data",
-                "      from the \"SELECTION:\" marker and may be some closing",
-                "      tags could be added to the mark-up",
-                "    - Data after the \"SELECTION:\" marker should correspond",
-                "      to the data selected in the browser",
-                "7) If the above requirements are met, the test is passed"
-            };
-
-        add(new DropPane());
-        Sysout.createDialogWithInstructions( instructions );
-
-        new ManualHTMLDataFlavorTest();
+    static final String INSTRUCTIONS = """
+        1) The test contains a drop-aware panel with a red background,
+        2) Open some page in a browser, select some text,
+           Drag and drop it on the red panel,
+           IMPORTANT NOTE: the page should be stored locally.,
+           otherwise for instance iexplore can prohibit drag and drop from,
+           the browser to other applications because of,
+           the protected mode restrictions.,
+           On Mac OS X do NOT use Safari, it does not provide the needed DataFlavor,
+        3) Check the data in the output area of this window,
+        5) The output should not contain information that any of,
+           flavors is not present in the system clipboard,
+        6) The output should contain data in three different formats,
+           provided by the system clipboard,
+            - Data after the "ALL:" marker should include the data,
+              from the "SELECTION:" marker",
+            - Data after the "FRAGMENT" marker should include the data,
+              from the "SELECTION:" marker and may be some closing,
+              tags could be added to the mark-up,
+            - Data after the "SELECTION:" marker should correspond,
+              to the data selected in the browser,
+        7) If the above requirements are met, the test is passed.
+    """;
+
+   static Frame createDropWindow() {
+        Frame frame = new Frame("Manual HTML DataFlavor Test");
+        frame.add(new DropPane());
+        frame.pack();
+        return frame;
```

Comment:
Changes looks good expect that when the the test is started and I switch to the browser window the drop target frame goes behind and becomes difficult to drag and drop to the Red panel.

Setting `frame.setAlwaysOnTop(true)` will ensure that the DnD panel is on top.

Reply:
done